src/components/CounterBadge/CounterBadge.visual.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { CounterBadge, COUNTER_BADGE_PRIORITY } from '.';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { classes } from './CounterBadge.visual.st.css';
+
+class CounterBadgeVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <CounterBadge {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'primary',
+        props: {
+          priority: COUNTER_BADGE_PRIORITY.primary,
+          value: 88,
+        },
+      },
+      {
+        it: 'secondery',
+        props: {
+          priority: COUNTER_BADGE_PRIORITY.secondary,
+          value: 13,
+          maximum: 9,
+        },
+      },
+      {
+        it: 'above maximum',
+        props: {
+          className: classes.palette,
```

Comment:
If you want the tests the style overrides then it should be a different test.
You can decide if to create tests for the style overrides or to remove the `CounterBadge.visual.st.css` and remove the className prop from the test.